pkg/instance/ttlDestroyConsumer.go

```diff
@@ -4,17 +4,22 @@ import (
 	"encoding/json"
 	"log"
 
+	"github.com/dhis2-sre/im-manager/pkg/config"
+	userClient "github.com/dhis2-sre/im-user/pkg/client"
+
 	"github.com/dhis2-sre/rabbitmq"
 	amqp "github.com/rabbitmq/amqp091-go"
 )
 
 type ttlDestroyConsumer struct {
+	config          config.Config
+	userClient      userClient.Client
 	consumer        *rabbitmq.Consumer
 	instanceService Service
 }
 
-func ProvideTtlDestroyConsumer(consumer *rabbitmq.Consumer, instanceService Service) *ttlDestroyConsumer {
-	return &ttlDestroyConsumer{consumer, instanceService}
+func ProvideTtlDestroyConsumer(config config.Config, userClient userClient.Client, consumer *rabbitmq.Consumer, instanceService Service) *ttlDestroyConsumer {
```

Comment:
I would only pass in username and password as that's the only thing that is needed from the config.